BioForm/profiles/templates/profiles/single_blog.html

```diff
@@ -0,0 +1,22 @@
+{% extends "profiles/base.html" %}
```

Comment:
It's better to name this file as just `blog` and for the blog list, you can use `blog_list`.